src/Kupo/App/Database.hs

```diff
@@ -74,6 +74,7 @@ import Numeric
 import qualified Data.Set as Set
 import qualified Data.Text as T
 import qualified Database.SQLite.Simple as Sqlite
+import Kupo.Data.Cardano (OutputReference)
```

Comment:
Small thing (which I should seemingly add to a 'coding standard' guidelines): qualified imports are separate from the rest and, [stylish-haskell](https://github.com/haskell/stylish-haskell) is used to automatically format imports (so that there's some form of consistency across the codebase).